docs/contributor/resources/01-kyma.md

```diff
@@ -1,4 +1,12 @@
-# Kyma Custom Resource
+# Kyma
+
+The `kymas.operator.kyma-project.io` Custom Resource Definition (CRD) is a comprehensive specification that defines the structure and format used to manage a cluster and its desired state. It contains the list of added modules and their state.
```

Comment:
Maybe strip the `is a comprehensive specification` part for all CRDs (including Kyma in user folder). It doesn't really say something I think. Just `[...] defines [...]` should be enough.